constructor/build_outputs.py

```diff
@@ -33,14 +34,34 @@ def process_build_outputs(info):
                 f"Available keys: {tuple(OUTPUT_HANDLERS.keys())}"
             )
         outpath = handler(info, **config)
-        logger.info("build_outputs: '%s' created '%s'.", name, os.path.abspath(outpath))
+        logger.info("build_outputs: '%s' created '%s'.", name, outpath)
+
+
+def dump_hash(info, algorithm=""):
+    if algorithm not in hashlib.algorithms_available:
+        raise ValueError(f"Invalid algorithm: {', '.join(algorithm)}")
```

Comment:
```suggestion
        raise ValueError(f"Invalid algorithm: {algorithm}")
```

I think you mean this?

Reply:
I originally planned to allow for multiple algorithms at once for one file. That didn't quite fit in the structure of how the outputs were handled (it only allowed for one file at a time). Now that I re-arranged it to output one hash file per installer, I added it back in.